java/java-jwt/security/audit/jwt-decode-without-verify.yaml

```diff
@@ -9,6 +9,9 @@ rules:
     cwe: 'CWE-345: Insufficient Verification of Data Authenticity'
     owasp: 'A2: Broken Authentication'
     source-rule-url: https://r2c.dev/blog/2020/hardcoded-secrets-unverified-tokens-and-other-common-jwt-mistakes/
+    category: security
+    technology:
+    - java-jwt
```

Comment:
Would this be `auth0.java-jwt` or the library identifier?

Reply:
I actually wonder if technology here should just be `jwt` since `java` is already captured by the language information. Or is `java-jwt` strictly different from `jwt`?

Reply:
@chmccreery I think this is applicable to a specific jwt library, `auth0.java-jwt`, so only applicable in the case you're using that, and then more broadly it's also `jwt` as the technology.